Gems/Atom/Feature/Common/Assets/Shaders/Materials/MultilayerPBR/StandardMultilayerPBR_Common.azsli

```diff
@@ -23,79 +33,6 @@
 #include "../MaterialInputs/ParallaxInput.azsli"
 #include "../MaterialInputs/UvSetCount.azsli"
 
-// ------ ShaderResourceGroup ----------------------------------------
-
-#define DEFINE_LAYER_SRG_INPUTS(prefix) \
-COMMON_SRG_INPUTS_BASE_COLOR(prefix)    \
-COMMON_SRG_INPUTS_ROUGHNESS(prefix)     \
-COMMON_SRG_INPUTS_METALLIC(prefix)      \
-COMMON_SRG_INPUTS_SPECULAR_F0(prefix)   \
-COMMON_SRG_INPUTS_NORMAL(prefix)        \
-COMMON_SRG_INPUTS_CLEAR_COAT(prefix)    \
-COMMON_SRG_INPUTS_OCCLUSION(prefix)     \
-COMMON_SRG_INPUTS_EMISSIVE(prefix)      \
-COMMON_SRG_INPUTS_PARALLAX(prefix)
-
-ShaderResourceGroup MaterialSrg : SRG_PerMaterial
-{
-    Texture2D m_blendMaskTexture;
```

Comment:
Where did this code get moved to? I had trouble finding it.

Reply:
Same as for all materials: The `MaterialParameter` - struct is generated from the parameters in the `StandardMultiLayerPBR.materialtype` and placed in `[project]/Cache/Intermediate Assets/materials/types/standardmultilayerpbr_parameters.azsli`.

The multilayer materialtype includes `StandardMultiLayerPBR_LayerProperties.json` three times with a different `shaderInputsPrefix`, which eventually results in e.g. `m_layer1_m_baseColor`, `m_layer2_m_baseColor` and `m_layer3_m_baseColor`. The `m_blendMaskTexture` is also very straighforward referenced in the materialtype, and also ends up in the struct.